src/common/src/config.rs

```diff
@@ -360,6 +360,16 @@ pub struct DeveloperConfig {
     /// The maximum size of the chunk produced by executor at a time.
     #[serde(default = "default::developer::stream_chunk_size")]
     pub stream_chunk_size: usize,
+
+    /// The initial permits that a channel holds, i.e., the maximum row count can be buffered in
+    /// the channel.
+    #[serde(default = "default::developer::stream_initial_permits")]
+    pub stream_initial_permits: usize,
```

Comment:
How about naming it `stream_exchange_initial_permits`?

Reply:
cool!